easybuild/easyconfigs/a/AlphaPulldown/AlphaPulldown-2.0.0-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,119 @@
+    ('PyTorch', '2.1.2', versionsuffix),
+    ('tqdm', '4.66.1'),
+    ('dm-tree', '0.1.8'),
+    ('py3Dmol', '2.1.0'),
+    ('HMMER', '3.4'),
+    ('HH-suite', '3.3.0'),
+    ('dm-haiku', '0.0.12', versionsuffix),
+    ('Uni-Core', '0.0.3', versionsuffix),
+    ('JupyterLab', '4.0.5'),
+]
+local_commit = 'cc4b0af60518c078305bbe4c584691d1ed9ade31'
+use_pip = True
+
+local_tests = [
+    'custom_db',
+    'remove_clashes_low_plddt',
+    'modelcif',
+    'features_with_templates',
+    'post_prediction',
+    # require pyrosetta, analysis aptainer image, and AlphaFold2 data:
+    # 'pdb_analyser',
+    # 'get_good_inter_pae',
+]
+local_testinstall_PATH = """ PATH=$(echo $PYTHONPATH|awk -F ':' '{print $1}')/../../../bin:$PATH """
+exts_list = [
+    ('contextlib2', '21.6.0', {
+        'checksums': ['ab1e2bfe1d01d968e1b7e8d9023bc51ef3509bba217bb730cee3827e1ee82869'],
+    }),
+    ('ml-collections', '0.1.1', {
+        'preinstallopts': "touch requirements.txt && touch requirements-test.txt && ",
+        'sources': ['ml_collections-%(version)s.tar.gz'],
+        'checksums': ['3fefcc72ec433aa1e5d32307a3e474bbb67f405be814ea52a2166bfc9dbe68cc'],
+    }),
+    ('PDBFixer', '1.9', {
+        'source_urls': ['https://github.com/openmm/pdbfixer/archive/'],
+        'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['88b9a77e50655f89d0eb2075093773e82c27a4cef842cb7d735c877b20cd39fb'],
+    }),
+    ('ihm', '1.3', {
+        'checksums': ['09f69809fd81509cc26b60253c55b02ce79fc01fc8f4a068bca2953a7dfd33be'],
+    }),
+    ('modelcif', '1.0', {
+        'checksums': ['e8375bc502a73dcfab0b7fbdd454d67d393bbb8969981eb52199d77192a3de56'],
+    }),
+    ('looseversion', '1.1.2', {
+        'checksums': ['94d80bdbd0b6d57c11b886147ba1601f7d1531571621b81933b34537cbe469ad'],
+    }),
+    ('mmtf-python', '1.1.3', {
+        'modulename': 'mmtf',
+        'checksums': ['12a02fe1b7131f0a2b8ce45b46f1e0cdd28b9818fe4499554c26884987ea0c32'],
+    }),
+    ('biopandas', '0.5.1.dev0', {
+        'checksums': ['6dc9de631babf8221c1ac60230133717039e08911f15e8ac48498c787022de12'],
+    }),
+    ('immutabledict', '4.1.0', {
+        'checksums': ['93d100ccd2cd09a1fd3f136b9328c6e59529ba341de8bb499437f6819159fe8a'],
+    }),
+    (name, version, {
+        'preinstallopts': "sed -i 's/[>=]=.*//g;s/tensorflow-cpu/tensorflow/g' setup.cfg && ",
+        'runtest': '%s pytest -s %s ' % (local_testinstall_PATH, " ".join('test/test_%s.py' % x for x in local_tests)),
+        'sources': [{
+            'filename': '%(name)s-%(version)s.tar.gz',
+            'git_config': {
+                'url': 'https://github.com/KosinskiLab',
+                'repo_name': 'AlphaPulldown',
+                'tag': version, 'recursive': True
+            }
+        }],
+        'testinstall': True,
+        'checksums': ['e338195987e003f3caadb06bda0ca56dece87e358738143ea72662f9ad69b1d4'],
```

Comment:
This needs to be [None], at least until EB v5 is out

Reply:
@akesandgren done